templates/js-cypress/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "cypress-module-api",
+  "version": "1.0.0",
+  "type": "module",
+  "description": "Run Cypress via its module API",
+  "engines": {
+    "node": ">=16.0.0"
+  },
+  "dependencies": {
+    "cypress": "^12.7.0",
+    "console.table": "^0.10.0",
+    "globby": "^13.1.3",
+    "apify": "^3.0.0"
+  },
+  "devDependencies": {
+    "@apify/eslint-config": "^0.3.1",
+    "eslint": "^8.20.0"
+  },
+  "scripts": {
+    "cypress:open": "../../node_modules/.bin/cypress open",
```

Comment:
If we keep cypress in dependencies, just `cypress open` should be enough.

Or maybe we could make `cypress` dev dependency, because on prod it should use the comand that is already in cypress image.

Reply:
keeping cypress open, deleting `"start": "node ./main.js"`

Reply:
actually keeping the start as well, otherwise, it cannot be run locally by `apify run`, I probably misunderstood it